client/state/form/shipping-label/actions.js

```diff
@@ -1,9 +1,13 @@
+import * as FormActions from 'state/form/actions';
+
 export const OPEN_PRINTING_FLOW = 'OPEN_PRINTING_FLOW';
 export const EXIT_PRINTING_FLOW = 'EXIT_PRINTING_FLOW';
 
-export const openPrintingFlow = () => ( {
-	type: OPEN_PRINTING_FLOW,
-} );
+export const openPrintingFlow = () => ( dispatch ) => {
+	dispatch( { type: OPEN_PRINTING_FLOW } );
+	dispatch( FormActions.setFormProperty( 'pristine', false ) );
+	dispatch( FormActions.nextStep() );
+};
```

Comment:
Like we talked about in chat this morning, this might be a good place to use https://github.com/tshelburne/redux-batched-actions - especially as you point out for rendering performance